src/js/hooks/cycleCount/useToResolveTab.jsx

```diff
@@ -0,0 +1,324 @@
+        </TableHeaderCell>
+      ),
+      cell: ({ getValue }) => (
+        <TableCell className="rt-td multiline-cell">
+          <div className="badge-container">
+            {getValue()}
+          </div>
+        </TableCell>
+      ),
+    }),
+    columnHelper.accessor((row) =>
+      row?.productCatalogs?.map((catalog) => <Badge label={catalog?.name} variant="badge--blue" key={catalog.id} />), {
+      id: 'productCatalogs',
+      header: () => (
+        <TableHeaderCell>
+          {translate('react.cycleCount.table.productCatalogue.label', 'Product Catalogue')}
+        </TableHeaderCell>
+      ),
+      cell: ({ getValue }) => (
+        <TableCell className="rt-td multiline-cell">
+          <div className="badge-container">
+            {getValue()}
+          </div>
+        </TableCell>
+      ),
+    }),
+    columnHelper.accessor('abcClass', {
+      header: () => (
+        <TableHeaderCell sortable columnId="abcClass" {...sortableProps}>
+          {translate('react.cycleCount.table.abcClass.label', 'ABC Class')}
+        </TableHeaderCell>
+      ),
+      cell: ({ getValue }) => (
+        <TableCell className="rt-td">
+          {getValue()}
+        </TableCell>
+      ),
+    }),
+    columnHelper.accessor('quantityOnHand', {
+      header: () => (
+        <TableHeaderCell sortable columnId="quantityOnHand" {...sortableProps}>
+          {translate('react.cycleCount.table.quantity.label', 'Quantity')}
+        </TableHeaderCell>
+      ),
+      cell: ({ getValue }) => (
+        <TableCell className="rt-td">
+          {getValue()}
+        </TableCell>
+      ),
+    }),
+  ], [currentLocale, sort, order]);
+
+  const emptyTableMessage = {
+    id: 'react.cycleCount.table.emptyTable.label',
+    defaultMessage: 'No products match the given criteria',
+  };
+
+  const exportTableData = () => {
+    spinner.show();
+    const date = new Date();
+    const [month, day, year] = [date.getMonth(), date.getDate(), date.getFullYear()];
+    const [hour, minutes, seconds] = [date.getHours(), date.getMinutes(), date.getSeconds()];
+    exportFileFromAPI({
+      url: CYCLE_COUNT_CANDIDATES(currentLocation?.id),
+      params: getParams({}),
+      filename: `CycleCountReport-${currentLocation?.name}-${year}${month}${day}-${hour}${minutes}${seconds}`,
+      afterExporting: spinner.hide,
+    });
+  };
+
+  const printResolveForm = () => {
+    // TODO: implement me!
+    console.log('print resolve form pressed');
+  };
+
+  const startResolution = async () => {
+    const payload = {
+      // TODO: replace this with a real call to startCount and pass the request id!
+      requests: checkedCheckboxes.map((productId) => ({
+        cycleCountRequest: productId,
+        countIndex: 1, // We only ever allow for a single recount, so index is always 1.
+      })),
+    };
+    spinner.show();
+    try {
+      await cycleCountApi.startCount(payload, currentLocation?.id);
```

Comment:
you can also change the name of the method here to probably `startResolution`

Reply:
it actually uses the same startCount API https://pihemr.atlassian.net/browse/OBPIH-6898